x-pack/plugins/observability/README.md

```diff
@@ -0,0 +1,27 @@
+# Observability plugin
+
+This plugin provides shared components and services for use across observability solutions, as well as the observability landing page UI.
+
+## Unit testing
+
+Note: Run the following commands from `kibana/x-pack/plugins/apm`.
```

Comment:
I think you meant `kibana/x-pack/plugins/observability` right?

Reply:
Copy/paste failure :)